infra/cardano-sl-infra.cabal

```diff
@@ -176,6 +176,7 @@ library
                       , safe-exceptions
                       , serokell-util >= 0.1.3.4
                       , stm
+                      , clock
```

Comment:
I believe you'll have to `./pkgs/generate.sh` and then commit the resulting `./pkgs/default.nix` in order for CI to pass.

Reply:
Indeed, that's what the logs advice too :). It worked locally, so it should work in CI as well.